src/test/java/HW21.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class HW21 extends BaseTest {
+
+    @Test
+     public void renamePlaylist() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.
![image](https://user-images.githubusercontent.com/104761482/212475073-e66b3252-9a99-420c-bfaa-54bd72a5bc6c.png)